kivy/garden/__init__.py

```diff
@@ -180,10 +180,13 @@ def load_module(self, fullname):
                 return self._load_module(fullname, moddir)
 
     def _load_module(self, fullname, moddir):
-        mod = imp.load_module(fullname, None, moddir,
-                              ('', '', imp.PKG_DIRECTORY))
+        spec = importlib.util.spec_from_file_location(fullname, moddir)
+        mod = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(mod)
         return mod
 
 
 # insert the garden importer as ultimate importer
 sys.meta_path.append(GardenImporter())
+
+
```

Comment:
I think removing these whitespaces should fix the PEP8 error

Reply:
Thanks! I made a new commit with the suggested change.